contributors/design-proposals/mount-options.md

```diff
@@ -0,0 +1,145 @@
+# Mount options for mountable volume types
+
+## Goal
+
+Enable Kubernetes users to specify mount options with mountable volumes
```

Comment:
To me, this should not be the goal at all. We should be allowing `kubernetes storage admins` to specify mount options. I actually object to `users` having anything to do with it.